services/search/main.py

```diff
@@ -0,0 +1,23 @@
+import testing.test_constants as constants
+from search_engine import SearchEngine
+
+def search_service(request):
+    """
+    Parses http json request and returns list of articles and their
+    scores depending on the input query
+    Args:
+        request: http POST body request as json
+    Returns:
+        list of articles with their respective scores
+    """
+    json = request.get_json()
+    query = json["query"]
+    search = SearchEngine(keywords_weight=2)
```

Comment:
The "2" should go in a constant, it can be helpful for testing and if anyone wants to use it somewhere else in the future. Otherwise, you end up with multiple hard-coded "2"s which can be hard to track down if someone wants to change it later on.